ChangeLog.md

```diff
@@ -11,6 +11,11 @@
 * Make HasSource and HasSink superclasses of HasState.
   See [#75](https://github.com/tweag/capability/pull/75)
 
+* Allow to retain capabilities in zoom and magnify.
+  See [#73](https://github.com/tweag/capability/pull/73)
+
+* Introduce the TypeOf type-family and capability short-cuts.
```

Comment:
```suggestion
* Introduce functional capabilities and the `TypeOf` type family.
```

I'm introducing a piece of terminology here. I'm using it in documentation that I will submit soon.